style.css

```diff
@@ -0,0 +1,67 @@
+html{
+    font-size: 15px;
+}
+img{
```

Comment:
should you be overriding img or creating a new class, img for header icons? img for table icons? Probably different sizes